README.md

```diff
@@ -17,6 +17,8 @@ The Mobile Center SDK uses a modular architecture so you can use any or all of t
 
 3. **Mobile Center Distribute**: Mobile Center Distribute will let your users install a new version of the app when you distribute it via the Mobile Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or postpone the new version. Once they choose to update, the SDK will start to update your application. This feature will NOT work if your app is deployed to the app store.
 
+4. **Mobile Center Push**: Mobile Center Push enables you to send push notifications to users of your app from the portal.
```

Comment:
It is from MicrosoftDoc for portal. Android doc is mentioning two SDKs (Firebase and Mobile Center) so it explicitly says `Mobile Center portal` but here, it doesn't need to be. Just copied from portal doc.